packages/wdio-sync/webdriverio-core.d.ts

```diff
@@ -816,6 +847,16 @@ declare namespace WebdriverIO {
         url(
             url?: string
         ): void;
+
+        /**
+         * This wait command is your universal weapon if you want to wait on something. It expects a condition
+         * and waits until that condition is fulfilled with a truthy value. If you use the WDIO testrunner the
+         * commands within the condition are getting executed synchronously like in your test.
+         */
+        waitUntil(
+            condition: Function,
```

Comment:
Do you think we can change it to `condition: () => boolean,` somehow?

Reply:
let me see if I can put this together, definitely makes sense!